plugins/woocommerce/src/Internal/Admin/ImportExport/HposOrderExportHandler.php

```
@@ -0,0 +1,244 @@
+		}
+
+		return true;
+	}
+
+	/**
+	 * When an order is imported via Tools > Import, backfill it into the HPOS table.
+	 *
+	 * @param int   $post_id The imported post ID.
+	 * @param array $original_post_id The original post object.
+	 */
+	public function maybe_backfill_hpos_order( $post_id, $original_post_id ) {
```

PHPMD (2.15.0)

Avoid unused parameters such as '$original_post_id'. (Unused Code Rules)

(UnusedFormalParameter)